popup-wrapper.html

```diff
@@ -3,7 +3,8 @@
 <head>
     <meta charset="UTF-8">
 </head>
-<body>
+<body style="font-family: sans-serif; padding: 0 .5em; white-space: nowrap">
```

Comment:
`white-space: nowrap` makes Chrome adapt the width of the popup to the content, otherwise the popup width is ~50px which looks ridiculous. I didn't want to hardcode a width either as I don't like the extra blank space around the error message